ui/src/components/ServiceMapModal/ServiceMapOptions.ts

```diff
@@ -1,83 +1,174 @@
+
+const minNodeScaling = 10
+const maxNodeScaling = 30
+
+const minEdgeScaling = 1
+const maxEdgeScaling = maxNodeScaling / 2
+
+const minLabelScaling = 11
+const maxLabelScaling = 16
+const selectedNodeColor = "#0C0B1A"
+const selectedNodeBorderColor = "#205CF5"
+const selectedNodeLabelColor = "#205CF5"
+const selectedEdgeLabelColor = "#205CF5"
+
+function customScaling(min, max, total, value) {
```

Comment:
can we change all functions to es6 syntax?
for example:
const customScaling = (min, max, total, value) => {